src/theme/index.js

```diff
@@ -1,7 +1,8 @@
 import _ from "lodash";
 import { colors, createMuiTheme, responsiveFontSizes } from "@material-ui/core";
 import typography from "./typography";
-import { softShadows } from "./shadows";
+//! Adds a error Material-UI: This elevation `4` is not implemented.
+// import { softShadows } from "./shadows";
```

Comment:
Please revert this, the shadows are not applied on the components, see the cards etc, the custom shadows are not applied. Have to look for some other way to fix it.

Reply:
@Abhishek-kumar09 Reverted.